sklearn/linear_model/base.py

```diff
@@ -360,6 +425,13 @@ class LinearRegression(LinearModel, RegressorMixin):
 
     normalize : boolean, optional, default False
         If True, the regressors X will be normalized before regression.
+        When the regressors are normalized, the fitted `coef_` are the same
```

Comment:
I agree this may not be the best wording. The discussion is in #5447 with @GaelVaroquaux. Let me know if you guys have a better and concise way to explain this property here.